TGC/common/event_modifiers.txt

```diff
@@ -7883,6 +7883,11 @@ Backed-Down-Manchuria = {
 
 ###End of German Extra Flavor ###
 
+###Netherlands Flavor###
+willem_i_absolutism = {}
```

Comment:
You sure you don't need an icon here?

Reply:
Not really no, in the future there will be some effect to them as well, but I think the red hammer it defaults to works just fine